test/streaming/get/test_network_consumer.py

```diff
@@ -249,6 +249,48 @@ async def client_test():
 
         await self.mock_server.validate(client_test, [StreamGrpc('getEquipmentForContainers', [_create_container_equipment_responses(feeder_network)])])
 
+    @pytest.mark.asyncio
+    async def test_get_equipment_for_container_supports_linked_containers(self, feeder_network: NetworkService):
+        async def client_test():
+            mor = (
+                await self.client.get_equipment_for_container(
+                    "f001",
+                    IncludedEnergizingContainers.INCLUDE_ENERGIZING_SUBSTATIONS,
+                    IncludedEnergizedContainers.INCLUDE_ENERGIZED_LV_FEEDERS
+                )
+            ).throw_on_error().value
+
+            assert len(mor.objects) == self.service.len_of(Equipment) == 3
+            _assert_contains_mrids(self.service, "fsp", "c2", "tx")
+
+        await self.mock_server.validate(client_test, [StreamGrpc('getEquipmentForContainers', [_create_container_equipment_responses(feeder_network)])])
+
+    @pytest.mark.asyncio
+    async def test_get_equipment_for_containers(self, feeder_network: NetworkService):
+        async def client_test():
+            mor = (
+                await self.client.get_equipment_for_containers(
+                    ["f001"],
+                    IncludedEnergizingContainers.INCLUDE_ENERGIZING_SUBSTATIONS,
+                    IncludedEnergizedContainers.INCLUDE_ENERGIZED_LV_FEEDERS
+                )
+            ).throw_on_error().value
+
+            assert len(mor.objects) == self.service.len_of(Equipment) == 3
+            _assert_contains_mrids(self.service, "fsp", "c2", "tx")
+
+        await self.mock_server.validate(client_test, [StreamGrpc('getEquipmentForContainers', [_create_container_equipment_responses(feeder_network)])])
+
+    @pytest.mark.asyncio
+    async def test_get_equipment_for_containers(self, feeder_network: NetworkService):
```

Comment:
todo: test name has already been used so replaces the above test. Please rename the tests to something appropriate.